rabbitmq/init.sls

```diff
@@ -1,6 +1,6 @@
 include:
-  - rabbitmq.install
-  - rabbitmq.plugins
-  - rabbitmq.os_tweaks
-  - rabbitmq.configure
-  - rabbitmq.permissions
+  - .install
+  - .plugins
+  - .configure
+  - .permissions
+  - .service
```

Comment:
We shouldn't include `.service` at the `init` level since it gets included in the configure and install states instead.